machine-learning/app/healthcheck.py

```diff
@@ -0,0 +1,13 @@
+import sys
+import requests
+import os
+
+port = os.getenv("IMMICH_PORT", 3003)
+
+try:
+  response = requests.get(f"http://localhost:{port}")
```

Comment:
The Node.js health check has a timeout set, but this one doesn't. They should probably be the same.